pkg/sentry/identity/kubernetes/validator.go

```diff
@@ -19,21 +19,19 @@ const errPrefix = "csr validation failed"
 
 var log = logger.NewLogger("dapr.sentry.identity.kubernetes")
 
-func NewValidator(client k8s.Interface, audiences []string, noDefaultTokenAudience bool) identity.Validator {
+func NewValidator(client k8s.Interface, audiences []string) identity.Validator {
 	return &validator{
 		client:    client,
 		auth:      client.AuthenticationV1(),
 		audiences: audiences,
-		// TODO: Remove once the NoDefaultTokenAudience feature is finalized
-		noDefaultTokenAudience: noDefaultTokenAudience,
 	}
 }
 
 type validator struct {
 	client    k8s.Interface
 	auth      kauth.AuthenticationV1Interface
 	audiences []string
-	// TODO: Remove once the NoDefaultTokenAudience feature is finalized
+	// TODO: Remove in Dapr 1.12 to enforce setting an explicit audience
 	noDefaultTokenAudience bool
```

Comment:
Should we remove it from here and simply set `true` in https://github.com/dapr/dapr/blob/d1a4c7945da5cb0087cfaac8531662156e84bee8/pkg/sentry/identity/kubernetes/validator.go#L57 ?

Reply:
Done